sodasql/dialects/postgres_dialect.py

```diff
@@ -94,10 +92,17 @@ def is_connection_error(self, exception):
         if exception is None:
             return False
         error_message = str(exception)
-        return error_message.find('Operation timed out') != -1
+        return error_message.find('Operation timed out') != -1 or \
```

Comment:
Are you sure you want to use string based error interpretation? This will fail if the locale is set to anything other than EN, I think. I would guess checking the proper sql state error code should be better. Assuming we are using PEP

Reply:
We will be checking this only in managed scan flow, where we are in control of the locale. 

It is a good remark, can you point me to the error code listing, I tried to find something which is a cross-driver standard but failed to find it

Reply:
The drivers should return the SQLSTATE error codes according to the standard. (https://en.wikipedia.org/wiki/SQLSTATE).

In the case of psycopg2 the exceptions/errors are in https://www.psycopg.org/docs/errors.html

Reply:
No they don't unfortunately, I am aware of that standard and already looked at this previously, I assumed there is something you could recommend that would work for psycopg2. Here is an example of raw exception (not a wrapper) of the psycopg2 error for connection/authentication (every time it is `OperationalError` without any properties/attributes - the only difference is in error message).

![image](https://user-images.githubusercontent.com/50055/108966125-167a4200-767e-11eb-9527-975cf4ce3deb.png)

Reply:
BTW, the place the exception is thrown is in binary library `_psycopg.cpython-<python_version>m-<arch>.so` so I can't get any more information about it